src/modules/userInteraction.js

```diff
@@ -0,0 +1,71 @@
+function generatePopup(info) {
+  return `
+    <div id = "modal">
+        <div id = "modal-content">
+            <img id = "close-btn" src = "../assets/closebtn.png" alt = "close button"/>
```

Comment:
- [ ] The close icon in the popup window is not showing as expected, it could be a path problem. Let us kindly resolve this